.gitignore

```diff
@@ -29,4 +29,4 @@ packages/k6/src/config/http.json
 
 # Secrets
 .envrc
-.env
+.env
```

Comment:
Hey, ive created a new PR since rebasing was gicing me issues on this branch. Here is the link please refer to this PR from now on. Thank you

https://github.com/smartcontractkit/external-adapters-js/pull/3529